apps/condo/domains/organization/integrations/sbbol/sync/syncBankAccounts.js

```diff
@@ -0,0 +1,88 @@
+const { getLogger } = require('@open-condo/keystone/logging')
+const { get, isEmpty } = require('lodash')
+
+const { initSbbolFintechApi } = require('@condo/domains/organization/integrations/sbbol/SbbolFintechApi')
+const { dvSenderFields } = require('@condo/domains/organization/integrations/sbbol/constants')
+const { BankAccount } = require('@condo/domains/banking/utils/serverSchema')
+const { RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
+const { ISO_CODES_FOR_SBBOL } = require('@condo/domains/common/constants/currencies')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+
+const logger = getLogger('sbbol/syncBankAccounts')
+
+/**
+ * Connects new BankAccount records for user according to accounts data from SBBOL.
+ *  @param {SbbolClientInfo} data
+ *  @param {String} bankIntegrationContextId
+ *  @param {Object} organization
+ */
+const _syncBankAccounts = async (data, bankIntegrationContextId, organization) => {
```

Comment:
Is there a reason to pass entire `SbbolClientInfo` or we can pass only accounts?
```suggestion
const _syncBankAccounts = async (accounts, bankIntegrationContextId, organization) => {
```